src/main/kotlin/ai/digital/integration/server/deploy/internals/cluster/operator/AzureAksHelper.kt

```diff
@@ -1,25 +1,254 @@
+                "az group list --query \"[?location=='$location']\" --output tsv | grep $groupName", throwErrorOnFailure = false, logOutput = false)
+        return result.contains(groupName)
+    }
+
+    private fun createResourceGroup(name: String, location: String, skipExisting: Boolean) {
+        val groupName = resourceGroupName(name)
+        var shouldSkipExisting = false
+        if (skipExisting) {
+            if (existsResourceGroup(groupName, location)) {
+                shouldSkipExisting = true
+            }
+        }
+        if (shouldSkipExisting) {
+            project.logger.lifecycle("Skipping creation of the existing resource group: {}", groupName)
+        } else {
+            project.logger.lifecycle("Create resource group: {}", groupName)
+            ProcessUtil.executeCommand(project,
+                    "az group create --name $groupName --location $location")
+        }
+    }
+
+    private fun deleteResourceGroup(groupName: String, location: String) {
+        if (existsResourceGroup(groupName, location)) {
+            project.logger.lifecycle("Create resource group: {}", groupName)
+            ProcessUtil.executeCommand(project,
+                    "az group delete --name $groupName --yes")
+        } else {
+            project.logger.lifecycle("Skipping delete of the resource group: {}", groupName)
+        }
+    }
+
+    private fun createCluster(name: String, clusterNodeCount: Property<Int>, clusterNodeVmSize: Property<String>, kubernetesVersion: Property<String>, skipExisting: Boolean) {
+        val groupName = resourceGroupName(name)
+        val clusterName = aksClusterName(name)
+        var shouldSkipExisting = false
+        if (skipExisting) {
+            val result = ProcessUtil.executeCommand(project,
+                    "az aks list --output tsv | grep $clusterName", throwErrorOnFailure = false, logOutput = false)
+            if (result.contains(clusterName)) {
+                shouldSkipExisting = true
+            }
+        }
+        if (shouldSkipExisting) {
+            project.logger.lifecycle("Skipping creation of the existing AKS cluster: {}", clusterName)
+        } else {
+            project.logger.lifecycle("Create AKS cluster: {}", clusterName)
+            var additions = ""
+            if (clusterNodeVmSize.isPresent) {
+                additions += " --node-vm-size \"${clusterNodeVmSize.get()}\""
+            }
+            if (kubernetesVersion.isPresent) {
+                additions += " --kubernetes-version \"${kubernetesVersion.get()}\""
+            }
+            ProcessUtil.executeCommand(project,
+                    "az aks create --resource-group $groupName --name $clusterName --node-count ${clusterNodeCount.getOrElse(2)} " +
+                            "--generate-ssh-keys --enable-addons monitoring $additions")
+        }
+    }
+
+    private fun connectToCluster(name: String) {
+        ProcessUtil.executeCommand(project,
```

Comment:
No need to check the output?
Maybe at least to print it would be good

Reply:
It is printing output in executeCommand and checking if it was success, in other case throwing exception